src/components/UserSignUp.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './component.css';
+import { Button, InputGroup, Input } from 'reactstrap';
+
+function UserSignup() {
+  const [FirstName, setFirstName] = useState('');
+  const [LastName, setLastName] = useState('');
+  const [EmailId, setEmailId] = useState('');
+  const [ContactNo, setContactNo] = useState('');
+  const [Password, setPassword] = useState('');
+  let User = {};
```

Comment:
@sayalisheth we don't need to let user here since it has no purpose outside submit, declare it there itself and use pascalCase in variable names.

Reply:
Done